contract/create_asset_response.go

```diff
@@ -0,0 +1,7 @@
+package contract
+
+import "github.com/vkhichar/asset-management/domain"
+
+type CreateAssetResponse struct {
```

Comment:
if we serialize this to JSON, this will become as following:
```
{
"object": {
     "id": "asset-uuid"
      ...
     }
}
```
however we need the following:
```
{
 "id": "asset-uuid"
 ...
}
```